pkg/controller/compliancesuite/compliancesuite_controller.go

```diff
@@ -241,13 +243,27 @@ func (r *ReconcileComplianceSuite) reconcileScanRemediations(suite *complianceop
 	var scanRemediations []*complianceoperatorv1alpha1.ComplianceRemediation
 
 	// Look for configMap with this scan label
-	listOpts := client.ListOptions{
-		LabelSelector: labels.SelectorFromSet(labels.Set{"compliance-scan": scan.Name}),
-	}
+	err := wait.PollImmediate(5*time.Second, 10*time.Minute, func() (bool, error) {
```

Comment:
lets use the global timeout variables here.

Reply:
this is the operator code, the purpose of the patch is to just check if it would help, in 'real' code I would fix this is the logcollector